boot/bootutil/src/bootutil_public.c

```diff
@@ -316,14 +316,30 @@ int
 boot_write_magic(const struct flash_area *fap)
 {
     uint32_t off;
+    uint32_t pad_off;
     int rc;
+    uint8_t magic[BOOT_MAGIC_ALIGN_SIZE];
+    uint8_t erased_val;   
```

Comment:
```suggestion
    uint8_t erased_val;
```
Dangling whitespaces